src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -39,7 +45,11 @@ export default function ProviderSelectionField({
   useEffect(
     () => {
       if (mounted && Object.keys(formData).length === 0) {
-        scrollAndFocus('.va-button-link');
+        if (showCCIterations) {
```

Comment:
@jbalboni The original implementation focuses on the residential/current location sort button, so I set the focus to the select in the cc iteration. If you think it makes more sense to focus on the first provider in the list I can update. I think it makes sense either way.

Reply:
I went back and looked at the what it's doing currently in master. It looks like it's 

- Focus on Choose provider h2 after you click "+ Choose a provider"
- When you click on "use your address/use current location", then focus jumps to loading spinner, then back to h2 when complete

I think doing both of those with the dropdown flow is what we want.

I don't see any intentional focusing on the sort buttons, though it does look like we're inadvertently doing that in this branch when the toggle is off (but not on master).

Reply:
It looks like the inadvertent focusing on the sort/dropdown buttons is due to my refactoring of the `CommunityCareProviderSelectionPage` to use `useSelector`. When I revert that change the focus works as intended, it looks like there is an extra render from updating the passed `formData` in the schemaform.

I fixed the focusing by removing a redundant useEffect in the `ProviderSelectionField` component.